src/components/core/circleGradient/CircleGradient.tsx

```diff
@@ -0,0 +1,34 @@
+type ColorVariants = {
+  peachLight: string;
+  peachDark: string;
+  lambadaLight: string;
+  lambadaDark: string;
+  gradient3Light: string;
+  gradient3Dark: string;
+  gradient4Light: string;
+  gradient4Dark: string;
+};
+
+const colorVariants: ColorVariants = {
+  peachLight: 'bg-peach-light',
+  peachDark: 'bg-peach-dark',
+  lambadaLight: 'bg-lambada-light',
+  lambadaDark: 'bg-lambada-dark',
+  gradient3Light: 'bg-gradient3-light',
+  gradient3Dark: 'bg-gradient3-dark',
+  gradient4Light: 'bg-gradient4-light',
+  gradient4Dark: 'bg-gradient4-dark',
+};
+
+interface CircleGradientProps {
+  color: keyof ColorVariants;
+}
+
```

Comment:
It's better to write the component here in the same format as we do in other parts of the project. Try to maintain consistency